src/ctia/task/migration/migrate_es_stores.clj

```diff
@@ -150,7 +150,7 @@
    services :- mst/MigrationStoreServices]
   (let [migrated (transduce migrations conj documents)
         {:keys [data errors]} (list-coerce migrated)
-        new-migrated-count (+ migrated-count (count data))]
+        new-migrated-count (+ (count data) migrated-count)]
```

Comment:
I suspect that the reduce hold the queue due to this ([from this discussion](https://stackoverflow.com/questions/44586291/how-to-forget-headgcd-for-lazy-sequences-in-clojure))

Reply:
it's not a very important remark but I think it would feel more natural if the state of the migrated count reside within the loop and not in the underlying function